.ci/scripts/set_default_linux_apps.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+set -e
+
+LOCAL_DESKTOP_FILE_DIR=~/.local/share/applications
+mkdir -p "$LOCAL_DESKTOP_FILE_DIR"
+CIPD_CHROME_DESKTOP_FILE=${LOCAL_DESKTOP_FILE_DIR}/cipd-chrome.desktop
+
+#MimeType=application/pdf;application/rdf+xml;application/rss+xml;application/xhtml+xml;application/xhtml_xml;application/xml;image/gif;image/jpeg;image/png;image/webp;text/html;text/xml;x-scheme-handler/http;x-scheme-handler/https;
+cat << EOF > "$CIPD_CHROME_DESKTOP_FILE"
```

Comment:
nit: I am not sure what the google style doc says but in shell I prefer to use {} with variable references. So 
$CIPD_CHROME_DESKTOP_FILE should be ${CIPD_CHROME_DESKTOP_FILE}.

Reply:
Fixed

Reply:
> nit: I am not sure what the google style doc says but in shell I prefer to use {} with variable references.

That is the Google Bash style preference as well.

Reply:
Awesome! Thanks Stuart.